airflow/utils/log/file_task_handler.py

```diff
@@ -476,16 +476,17 @@ def _init_file(self, ti):
             # if this is true, we're invoked via set_context in the context of
             # setting up individual trigger logging. return trigger log path.
             full_path = self.add_triggerer_suffix(full_path=full_path, job_id=ti.triggerer_job.id)
-        self._prepare_log_folder(Path(full_path).parent)
+        path = Path(full_path)
+        self._prepare_log_folder(os.fspath(path.parent))
```

Comment:
Doesn’t `_prepare_log_folder` take a Path instead? The `os.fspath` call shouldn’t be there.